Library/Homebrew/keg_relocate.rb

```diff
@@ -72,13 +72,14 @@ def text_files
     # file with that fix is only available in macOS Sierra.
     # http://bugs.gw.com/view.php?id=292
     with_custom_locale("C") do
-      path.find do |pn|
-        next if pn.symlink? || pn.directory?
-        next if Metafiles::EXTENSIONS.include? pn.extname
-        if Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text") ||
-           pn.text_executable?
-          text_files << pn
-        end
+      files = path.find.reject { |pn|
```

Comment:
We have `Style/BlockDelimiters` set to `semantic`, which enforces curly braces around functional blocks and `do...end` around procedural blocks. If you'd rather we just use `do...end` for all multiline blocks, we can change that in a separate PR, as it will require many more changes elsewhere.